docs/source/tutorials/file.md

```diff
@@ -39,10 +39,23 @@ bucket2 = endpoint2
 
 Then run the following command, the config file will be read by default to authorize oss.
 
-~~~python
+```python
 from easycv.file import io
 io.access_oss()
-~~~
+```
+
+**Method3:**
+
+Set environment variables as follow,  EasyCV will automatically parse environment variables for authorization:
```

Comment:
add example for setting config file path

Reply:
done!